CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 0.2.1 (2018-08-16)
```

Comment:
Should the system file fix you made be included here?

Reply:
don't think so, it's a fairly minor bugfix

Reply:
Fair enough.